parsers/handleWordpressComponents.js

```diff
@@ -74,6 +74,56 @@ export function fixAccordions() {
   };
 }
 
+const findCaptionText = (figureCaption, key) => {
+  let figCaptionValue = '';
+  const nestedFigCap = figureCaption.filter((o) => o.type === key);
```

Comment:
That name is actually lying - these are the nodes in `figureCaptionNodes` that are of type `'text'` - so they are in fact not nested, they are as `shallow` as it gets :) 

`figureCaptionTextNodes`?

Reply:
That makes sense, will add that change to :)